install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/tasks/translations.yml

```diff
@@ -1,8 +1,10 @@
 ---
 
 - name: Install SecureDrop Python requirements in virtualenv for translation work
-  shell: |
-    python3 -m venv /tmp/securedrop-app-code-i18n-ve
+  shell: >
+    set -e &&
+    python3 -m venv /tmp/securedrop-app-code-i18n-ve &&
+    /tmp/securedrop-app-code-i18n-ve/bin/pip3 install "setuptools-scm==5.0.2" &&
```

Comment:
Note that hash checking is not performed on this dependency, but that was already the case, as it was being installed as a build dependency via python-dateutil's `setup.py.`